### DISCUSSION TITLE
#support

### DISCUSSION THREAD

**Discussion by Irwan1708** [blockscout/blockscout#10065] [Support]
My wallet was accidentally deleted, and I haven't had time to backup the private key. Can you help me? Please...😭😭😭

**Comment by rojob420** [comment]
> My wallet was accidentally deleted, and I haven't had time to backup the private key. Can you help me? Please...😭😭😭

**Comment by rojob420** [comment]
My wallet was accidentally deleted, and I haven't had time to backup the private key. Can you help me? Please...😭😭😭

**Comment by sl1depengwyn** [comment]
This repo is for Blockscout-related questions. Blockscout is an open source blockchain explorer. Thus, your question is out of scope, where the Blockscout team can help, you better contact the chain support directly.
